pkg/storage/tableland/impl/tableland/store.go

```diff
@@ -69,7 +74,11 @@ func (s *Store) InsertParts(ctx context.Context, parts []local.Part) error {
 	if err != nil {
 		return fmt.Errorf("getting sql to insert parts: %v", err)
 	}
-	return s.writeSQL(ctx, sql)
+	hash, err := s.writeSQL(ctx, sql)
+	if err != nil {
+		return fmt.Errorf("writing SQL: %v", err)
+	}
+	return s.trackTxn(ctx, hash, "parts", sql)
```

Comment:
All the changes in here are to tack the txn info when appropriate, which is anywhere we're inserting data into our Tableland tables.